_i18n/fr/resources/moneropedia/block.md

```diff
@@ -1,16 +1,15 @@
 ---
-entry: "Block"
-terms: ["block", "blocks"]
-summary: "a container of transactions, a sequence of which forms a blockchain"
+entry: "Bloc"
+terms: ["block", "blocks", "bloc", "blocs"]
+summary: "Un conteneur de transactions, une séquence qui forme la chaîne de blocs."
```

Comment:
une séquence qui forme la chaîne de blocs --> une séquence __desquelles__ forme une chaîne de blocs

(ou quelque chose comme ca)

Reply:
"un conteneur de transactions, dont une séquence forme une chaîne de blocs" ?

Reply:
Euhhh .. l'important c'est que ce n'est pas une séquence de transactions qui forme une chaîne de blocs. C'est une séquence de blocs qui forme la chaîne de blocs, évidemment. C'est pas bien écrit en Anglais je crois. On entend que c'est cela que veut dire l'écrivain, mais il faut déjà le comprendre.

Does that make sense?  :-)

Reply:
Oui, c'est bien dans ce sens que j'ai corrigé.

Reply:
Is it unclear?

Reply:
I think your suggestion is fine.